students/erica_edwards/Lesson03/mailroom.py

```diff
@@ -0,0 +1,97 @@
+
+from operator import itemgetter
+
+Donors = [('Sandy Pie', [75]),
+          ('Judy Smith', [75, 100, 1000]),
+          ('Mike Jones', [75, 1000]),
+          ('Joe Smith', [75, 100, 2000]),
+          ('Kelly Blue', [75, 150, 275])]
+
+def menu():
+    quit = False
+    while not quit:
+        print()
+        print('''Menu:   
+                    1. Send a Thank You.
+                
+                    2. Create a Report.
+            
+                    3. Quit.''')
+        print()
+        response = input("Please enter a number to make your selection. ")
+        if response == '1':
+            donor_name()
```

Comment:
function name is a bit confusing and doesn't tell the reader what the function does?

`send_thank_you` would be more suitable